airbyte/progress.py

```diff
@@ -118,7 +124,7 @@ def _get_elapsed_time_str(seconds: float) -> str:
     return f"{hours}hr {minutes}min"
 
 
-class ReadProgress:
+class ProgressTracker:  # noqa: PLR0904  # Too many public methods
```

Comment:
Change note: refactored `ReadProgress` as `ProgressTracker`, applicable to both reads and writes.